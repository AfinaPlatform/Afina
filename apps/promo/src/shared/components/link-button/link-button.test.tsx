import LinkButton from "./link-button";
import { render } from "@testing-library/react";

describe("shared/link-button", () => {
  it("render a html tag", () => {
    const { container } = render(<LinkButton href="/">Click</LinkButton>);
    expect(container.querySelector("a")?.tagName).toBe("A");
  });

  it("snapshot", () => {
    const { asFragment } = render(<LinkButton href="/">Click</LinkButton>);
    expect(asFragment()).toMatchSnapshot();
  });
});
