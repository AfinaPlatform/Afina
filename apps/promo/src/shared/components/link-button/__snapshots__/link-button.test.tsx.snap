// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/link-button snapshot 1`] = `
<DocumentFragment>
  <a
    class="root"
    href="/"
  >
    Click
  </a>
</DocumentFragment>
`;
